server/api/impl/aws/provision_data.go

```diff
@@ -111,9 +111,13 @@ func getProvisionData(b []byte, logger log) (*ProvisionData, error) {
 				Key:  assetName,
 				File: assetFile,
 			})
-			logger.Debug("Found function bundle", "file", file.Name)
+			if logger != nil {
```

Comment:
Can that case ever happen?

Reply:
Yes, function `aws.GetProvisionDataFromFile` could be called with `nil`